### ISSUE TITLE
Wrong install command in the readme

### ISSUE THREAD

**Issue by verbeeckjan** [tjwoon/csZBar#38]
The readme says: 
`cordova plugin add org.cloudsky.cordovaplugins.zbar`

However that doesn't work, the following does work however, assuming this is the same project as this, it should be changed in the readme.
`cordova plugin add org.madmobile.cordovaplugins.zbarscanner`

**Comment by Frosty2** [comment]
I've noticed the same issue.

**Comment by bau720123** [comment]
+1

**Comment by wesleywesley** [comment]
+1

newbies won't be able to install it.. please fix the readme

**Comment by tjwoon** [comment]
I think the plugin failed to install with newer versions of the Cordova CLI tools because I did not migrate this plugin to NPM. This has been implemented in [9769ad98889145fb4b31704a8337c28fc449ddc1] and [fbb4a31d39a9c898cc22ddd262a509899c3afb07].

New installation instructions are now in the [README](https://github.com/tjwoon/csZBar#readme). The README in NPM is outdated. Will be fixed in the next update.
